src/Microsoft.Azure.WebJobs.Extensions.Kafka/Serialization/LocalSchemaRegistry.cs

```diff
@@ -43,26 +43,37 @@ public Task<List<string>> GetAllSubjectsAsync()
             throw new System.NotImplementedException();
```

Comment:
I think we don't need to throw here. I know this is not a changed you did.

Reply:
Done. Could you have a look at the change? 
https://github.com/Azure/azure-functions-kafka-extension/pull/151/commits/af6f1dc12f56cdd10f1d6e104beb7050d3abd614